torch/distributed/optim/optimizer.py

```diff
@@ -154,10 +154,10 @@ class DistributedOptimizer:
 
     `DistributedOptimizer` creates the local optimizer with TorchScript enabled
     by default, so that optimizer updates are not blocked by the Python Global
-    Interpreter Lock (GIL) during multithreaded training (e.g. Distributed Model
-    Parallel). This feature is currently in beta stage, enabled for optimizers
-    including `Adagrad`, `Adam`, `SGD`, `RMSprop`, `AdamW` and `Adadelta`. We
-    are increasing the coverage to all optimizers in future releases.
+    Interpreter Lock (GIL) in the case of multithreaded training (e.g. Distributed
+    Model Parallel). This feature is currently enabled for most optimizers. You
+    can also follow the recipe in PyTorch tutorials to enable TorchScript support
```

Comment:
Nit: link the recipe? :)

Reply:
I tried but I don't know how I can link it though.. since the recipe don't have a dev page and it will be published together with 1.9, so we don't have a link now.. Do you know how we could get a link before the release? Or should we link a deploy preview as [this](https://deploy-preview-1465--pytorch-tutorials-preview.netlify.app/) or simply the PR link?